docs/src/components/v1/migration-box.tsx

```diff
@@ -0,0 +1,46 @@
+import type { ReactNode } from "react";
+import { Button } from "@/components/ui/button";
+import {
+  CollapsibleContent,
+  Collapsible as CollapsibleRoot,
```

Comment:
Is there a reason we need this rename?
```suggestion
  Collapsible,
```